backend/api/users.py

```diff
@@ -102,19 +102,19 @@ def get_user(mail):
             res: A user with given mail as json
     """
     res = g.user_datastore.get_user(mail)
-    if res is None:
+    if not res:
         return make_response("Unknown User with Email-address: " + mail, 400)
 
     return jsonify(res)
 
 
 @users.route('/api/users/bookmarks/<uuid:id>', methods=['POST'])
 @login_required
-def insert_bookmarks(id):
-    user = current_user
-    res = g.user_datastore.get_user(user['email'])
-    if res is None:
-        return make_response("Unknown User with Email-address: ", 400)
+def insert_bookmark(id):
+    res = g.user_datastore.get_user(current_user.email.to_python())
```

Comment:
is this even necessary? Can't we just use res = current_user? I don't see why not, get_user and current_user are both of type User(Document, UserMixin)